qir/qsharp/Atomics/AdvancedConstantFolding/QSharpVersion/Example.csproj

```
@@ -1,8 +1,8 @@
-<Project Sdk="Microsoft.Quantum.Sdk/0.23.195983">
+<Project Sdk="Microsoft.Quantum.Sdk/0.24.201332">
```

(nit) Consider either removing the version (see below) or changing to `0.24.210930`.
```suggestion
<Project Sdk="Microsoft.Quantum.Sdk">
```

---

All versions was updated on main and latest merge should address this.